lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
 const Tab = props => {
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
   return (
     <div
-      className={''}
+      className={'tab'}
```

Comment:
Another great spot for a ternary. One that returns the string `tab` or `tab active-tab` depending on whether or not the tab prop matches the selectedTab prop.